### ISSUE TITLE
[Sharing-NG] multiple mount request for items is allowed

### ISSUE THREAD

**Issue by S-Panta** [owncloud/ocis#8735]
## Describe the bug
Once an item is mounted via API, it retuns `201` status code as specified in swagger with body content of drive item. But,  hitting API a second time, it still returns `201` status code but nothing get returned in body

## Steps to reproduce
1.Login to ocis as `admin`
2.create a folder `test` and share to `einstein`
3. click disable sync.
4. Send API request to mount resource
```
curl -X POST 'https://host.docker.internal:9200/graph/v1beta1/drives/{share-Space-Id}/root/children'  
-d '{
    "remoteItem": {
        "id":"'123"
    }
}' -ueinstein:relativity
```
5. It will mount the item. Send API request again

## Actual behavior
The status code is 201 with empty body

Same is true while unmounting resource.

**Comment by S-Panta** [comment]
@rhafer

**Comment by 2403905** [comment]
Fixed by PR the [ #8817](https://github.com/owncloud/ocis/pull/8817)
